src/resources/app-en.json

```
@@ -1309,6 +1310,15 @@
     "placeholders": {
       "opt_in": "Text",
       "opt_out": "Text"
+    },
+    "chipLabels": {
+      "active": "Active",
+      "available": "Premium"
+    },
+    "modal": {
+      "title": "Request Health Record Data",
+      "content": "<0>Now it is possible to import participants' Electronic Health Care Data, and use that data with what you collect in your Mindlogger Activities.</0> <1>Please check out the website for more information.</1>",
```

I know the design says `Mindlogger`, but the `L` should always be capitalized (till we rebrand 😛).
```suggestion
      "content": "<0>Now it is possible to import participants' Electronic Health Care Data, and use that data with what you collect in your MindLogger Activities.</0> <1>Please check out the website for more information.</1>",
```

---

Updating 🤓